### PR TITLE
Using Variables

### PR DESCRIPTION
**Pull request by Walentino**

### CODE REVIEW

File: terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,7 @@
+env_code = "orlando"
+
+vpc_cidr = "10.0.0.0/16"
+
+public_cidr = ["10.0.0.0/24", "10.0.1.0/24", "10.0.2.0/24"]
+
+private_cidr = ["10.0.100.0/24", "10.0.101.0/24", "10.0.103.0/24"]
```
Comment (vladimirmukhin) [inline terraform.tfvars]: Please add trailing newline

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,7 @@
+variable "env_code" {}
+
+variable "vpc_cidr" {}
+
+variable "public_cidr" {}
+
+variable "private_cidr" {}
```
Comment (vladimirmukhin) [inline variables.tf]: Please add trailing newline

File: vpc.tf [hunk 3]
```diff
@@ -1,51 +1,42 @@
+locals {
+  public_cidr = ["10.0.0.0/24","10.0.1.0/24","10.0.2.0/24"]
+  private_cidr = ["10.0.100.0/24","10.0.101.0/24","10.0.103.0/24"]
+  availability_zones = ["us-east-1a", "us-east-1b","us-east-1c"]
```
Comment (vladimirmukhin) [inline vpc.tf]:
```suggestion
  public_cidr        = ["10.0.0.0/24","10.0.1.0/24","10.0.2.0/24"]
  private_cidr       = ["10.0.100.0/24","10.0.101.0/24","10.0.103.0/24"]
  availability_zones = ["us-east-1a", "us-east-1b","us-east-1c"]
```

equal signs must be aligned, you can also run terraform fmt to have this resolved automatically

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

**Comment by Walentino** [comment]
Please review and approve.

File: terraform.tfvars (hunk 1 above)
Comment (Walentino) [reply terraform.tfvars]: I have added trailing newline.

File: variables.tf (hunk 2 above)
Comment (Walentino) [reply variables.tf]: I have added trailing newline.

**Pull request merged**